Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/CompoundBocListValidationFailureHandlerTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Moq;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.ObjectBinding.Web.UI.Controls.Validation;
+
+namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls.Validation
+{
+  [TestFixture]
+  public class CompoundBocListValidationFailureHandlerTest
+  {
+    [Test]
+    public void HandleValidationFailures_CallsContainedHandlers ()
+    {
+      var handler1 = new Mock<IBocListValidationFailureHandler>();
+      var handler2 = new Mock<IBocListValidationFailureHandler>();
```

Comment:
```suggestion
      var handler1Mock = new Mock<IBocListValidationFailureHandler>();
      var handler2Mock = new Mock<IBocListValidationFailureHandler>();
```
(sadly we don't already suffix "mock" or "stub", but we should)